README.md

```diff
@@ -0,0 +1,209 @@
+# Digital Commerce Core Reference Template
+
+This is the base reference application for the conga ecommerce product. Follow the below instructions to get started. See the [docs](https://cmoyle336.github.io/sdk-docs/overview.html) for more detailed instructions on interacting with the underlying SDK.
+
```

Comment:
Please use below link https://apttus-ecom.github.io/December21/index.html as reference, the one mentioned is old referencing apttus libraries.